modules/op-configuring-pipelines-as-code-logging.adoc

```diff
@@ -0,0 +1,124 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-pipelines-as-code-logging_{context}"]
+= Configuring {pac} logging 
+
+You can configure {pac} logging with the `pac-config-logging` config map located in the same namespace as the controllers. The default namespace is `openshift-pipelines`.
```

Comment:
IMHO since it's not possible to configure namespace in downstream, we could merge those two sentences `... config map located in the namespaces openshift-pipelines`.